bin/adam-submit

```diff
@@ -22,40 +22,15 @@ SCRIPT_DIR="$(cd `dirname $0`/..; pwd)"
 
 # Get list of required jars for ADAM
 ADAM_JARS=$("$SCRIPT_DIR"/bin/compute-adam-jars.sh)
+ADAM_CLI_JAR=$(echo "$ADAM_JARS" | tail -n 1)
 
+ADAM_JARS=$(echo "$ADAM_JARS" | tail -n +1 | perl -pe 's/\n/,/ unless eof')
```

Comment:
Yea, I was missing that you had that `grep` method in this file previously; it's certainly nice to see it deleted here.

I think there are two orthogonal questions:

1. find the CLI jar via the assumption that appassembler puts it last on the classpath vs. by `grep`ing.
2. do the above in `adam-submit` vs. in `compute-adam-jars.sh`.

I'm ok with either approach to 1), but am somewhat interested in moving 2) to `compute-adam-jars.sh`, that way `adam-submit` and `adam-shell` both save a `| perl -pe 's/\n/,/ unless eof'`, and we can create a cleaner interface to the `| head -n -1` that is currently here in `adam-submit`.

Anyways, the way you have it wfm too, just articulating the options I see.